deathcause.lua

```diff
@@ -1,4 +1,5 @@
 -- show death cause of a creature
+--@ module = true
```

Comment:
since you are exposing this as a module, please mark everything that *shouldn't* be exposed as `local`